include/nanogui/glcanvas.h

```diff
@@ -21,47 +25,67 @@ NAMESPACE_BEGIN(nanogui)
 
 /**
  * \class GLCanvas glcanvas.h nanogui/glcanvas.h
- * \brief Canvas widget for rendering OpenGL content
+ *
+ * \brief Canvas widget for rendering OpenGL content.  This widget was
+ *        contributed by Jan Winkler.
  *
  * Canvas widget that can be used to display arbitrary OpenGL content. This is
  * useful to display and manipulate 3D objects as part of an interactive
  * application. The implementation uses scissoring to ensure that rendered
  * objects don't spill into neighboring widgets.
  *
- * Usage: override `drawGL` in subclasses to provide custom drawing code.
+ * \rst
+ * **Usage**
```

Comment:
For the most part, all of @fairlight1337 's comments stayed the same.  I just linked back to the example, and put some `.`'s at the end of doc comments because I've been obsessively doing that to everything.  AKA these doc changes are just for consistency, but I also added that he contributed the widget :)

Reply:
Sounds good :)